activesupport/lib/active_support/testing/file_fixtures.rb

```diff
@@ -0,0 +1,35 @@
+module ActiveSupport
+  module Testing
+    # Adds simple access to sample files called file fixtures.
+    # File fixtures are normal files stored in
+    # <tt>ActiveSupport::TestCase.file_fixture_directory</tt>.
+    module FileFixtures
+      extend ActiveSupport::Concern
+
+      included do
+        class_attribute :file_fixture_directory
```

Comment:
There's no big deal but I guess we can also pluralize "fixture" here (so we are consistent with the default generated path), what do you think ? Otherwise, awesome addition, this was really something missing ! :+1: